docs/monitoring/08-01-overview.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Overview
+type: Tutorials
+---
+
+The set of monitoring tutorials you are about to read describes the complete monitoring flow for your metrics in Kyma. Going through the tutorials, you will get the gist of Kyma in-built monitoring applications, such as Prometheus, Grafana, and Alertmanager. You will also understand how and where you can observe and visualize your service metrics to monitor them for any alerting values.
+
+All tutorials use the [`monitoring-custom-metrics`](https://github.com/kyma-project/examples/tree/master/monitoring-custom-metrics) example and one of its services called `sample-metrics-8081`. This service exposes the `cpu_temperature_celsius` custom metric on the `/metrics` endpoint. This custom metric is the central element of all tutorials. Its value simulates the current processor temperature and changes randomly from 60 to 90 degrees Celsius. The alerting threshold in these tutorials is 75 degrees Celsius. If the temperature exceeds this value, the Grafana dashboard, Prometheus rule, and Alertmanager notifications you create clearly inform you of the alerting value.
```

Comment:
```suggestion
All tutorials use the [`monitoring-custom-metrics`](https://github.com/kyma-project/examples/tree/master/monitoring-custom-metrics) example and one of its services called `sample-metrics-8081`. This service exposes the `cpu_temperature_celsius` custom metric on the `/metrics` endpoint. This custom metric is the central element of all tutorials. Its value simulates the current processor temperature and changes randomly from 60 to 90 degrees Celsius. The alerting threshold in these tutorials is 75 degrees Celsius. If the temperature exceeds this value, the Grafana dashboard, Prometheus rule, and Alertmanager notifications you create clearly inform you about this.
```
I wouldn't say that they inform you about the value itself (although it's mentioned), but about the fact that the temp value was exceeded.